app/components/chart/daily.js

```diff
@@ -33,6 +33,7 @@ var Daily = React.createClass({
   propTypes: {
     bgPrefs: React.PropTypes.object.isRequired,
     chartPrefs: React.PropTypes.object.isRequired,
+    timePrefs: React.PropTypes.object.isRequired,
```

Comment:
Basically because there are now a couple places in app.js where they might be modified - via the query params or after fetch of data (when we check most recent upload metadata for the timezone), so it was easier to break them out than to keep deep-cloning `chartPrefs` and updating them there.